devtools/common/src/main/java/io/quarkus/cli/commands/writer/FileWriter.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * 
+ */
+package io.quarkus.cli.commands.writer;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * @author guillaumedufour
```

Comment:
Probably best to remove this, we aren't really using it elsewhere

Reply:
ok sorry it's by default in eclipse

Reply:
np :)